packages/core/utils/entityTypes.ts

```diff
@@ -33,6 +33,12 @@ export interface TokenAccount {
 	quotes?: Record<string, number>;
 	balance: string;
 	decimals: number;
+
+	/**
+	 * This attribute is used for tezos token identifier
+	 * (combine with address - smart contract address)
+	 * */
+	tokenId?: number;
```

Comment:
then should name it `tezosTokenId` instead I guess

Reply:
must discuss further about this, we want unified Interface for different networks while still respect their nature.

Reply:
will rename later